acceptance-tests/upgrade/upgrade_suite_test.go

```diff
@@ -2,15 +2,21 @@ package upgrade_test
 
 import (
 	"flag"
+	"fmt"
+	"os"
 	"testing"
 
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
+
+	"csbbrokerpakaws/acceptance-tests/helpers/brokers"
 )
 
 var (
-	developmentBuildDir string
-	releasedBuildDir    string
+	developmentBuildDir   string
+	releasedBuildDir      string
+	extraEnvOptions       = make([]brokers.Option, 0)
```

Comment:
I think there's some syntax confusion going on here. This makes an **array** wheres in Go it's much more common to use a **slice**. The syntax for that is:
```go
var extraEnvOptions []brokers.Option
```